modules/doc/content/training/more_detail/MOOSE_2024_10_15-17_UIUC.md

```diff
@@ -6,7 +6,26 @@
 
 !include training/more_detail/generic_moose_training.md
 
-#### Location Details
+#### Getting to Urbana-Champaign
+
+!style! halign=left
++Flying:+ There are three commercial options:
+!style-end!
+
+- Champaign-Urbana (Willard Airport), located 15 minutes from campus. Direct flights to:
+    - Chicago
+    - Dallas
+- Central Illinois Regional Airport (Bloomington), located 1 hour from campus. Direct flights to:
+    - Chicago
+    - Dallas
+    - Atlanta
+    - Detroit
+    - Orlando
+- O'Hare International Airport (Chicago), located 2.5 hours from campus
```

Comment:
```suggestion
- Champaign-Urbana (Willard Airport), located 15 minutes from campus. Direct flights to:

    - Chicago
    - Dallas

- Central Illinois Regional Airport (Bloomington), located 1 hour from campus. Direct flights to:

    - Chicago
    - Dallas
    - Atlanta
    - Detroit
    - Orlando
    
- O'Hare International Airport (Chicago), located 2.5 hours from campus
```

Apologies - one more thing. These nested lists require some empty lines for docs to pick them up as nested.